package.json

```diff
@@ -14,7 +14,7 @@
   "devDependencies": {
     "tslint": "^5.4.3",
     "typedoc": "^0.7.1",
-    "typescript": "~2.3.4",
+    "typescript": "^2.5.2",
```

Comment:
please use a ~ dependency, TS often breaks in minors. Also update peerDep

Reply:
Might wanna add tslint to your CI 😛